### ISSUE TITLE
Not working with latest version of langchain==0.0.158

### ISSUE THREAD

**Issue by davidmokos** [amosjyng/langchain-visualizer#35]
Seems that langchain refactored the callbacks in https://github.com/hwchase17/langchain/commit/d3ec00b5664798be74daa9bdc28ae7c2c86ee22e

Hot fix is to remove the "callbacks" argument from kwargs in hijacking.py file:

```
def get_overridden_call(viz_cls, og_method_name):
    def overridden_call(og_self, *args, **kwargs):
        """Preserve sync nature of OG call method"""
        ice_agent = get_viz_wrapper(viz_cls, og_self, og_method_name)
        if (
            not hasattr(og_self.__class__, "_should_trace")
            or og_self.__class__._should_trace
        ):
            # ICE displays class name in visualization
            ice_agent.__class__.__name__ = og_self.__class__.__name__
            if "callbacks" in kwargs.keys():
                del kwargs["callbacks"]

            return asyncio.get_event_loop().run_until_complete(
                ice_agent.run(*args, **kwargs)
            )

        return ice_agent.og_fn(og_self, *args, **kwargs)

    return overridden_call
```

**Comment by vnktsh** [comment]
@davidmokos Thanks for the hot fix, saved a ton of time debugging

**Comment by amosjyng** [comment]
Thanks for bringing this to my attention! This should now be fixed in the visualizer v0.0.21
